tests/testsuite/namespace.go

```diff
@@ -375,10 +377,11 @@ func CalculateNamespaces() {
 	NamespaceTestDefault = fmt.Sprintf("%s%d", NamespaceTestDefault, worker)
 	NamespaceTestAlternative = fmt.Sprintf("%s%d", NamespaceTestAlternative, worker)
 	NamespacePrivileged = fmt.Sprintf("%s%d", NamespacePrivileged, worker)
+	NamespaceMigrationTarget = fmt.Sprintf("%s%d", NamespaceMigrationTarget, worker)
```

Comment:
NamespaceTestAlternative should be just as good for your purposes, unless I'm missing something

Reply:
> Unless that is using another runner and the namespaces do not collide.

:point_up: